pallets/ds-maps/src/lib.rs

```diff
@@ -0,0 +1,183 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+#[cfg(feature = "std")]
+use serde::{Deserialize, Serialize};
+
+use frame_support::{
+    codec::{Decode, Encode},
+    decl_error, decl_event, decl_module, decl_storage, dispatch, ensure,
+    weights::{Weight},
+    Parameter,
+};
+use frame_system::ensure_signed;
+use pallet_ds_accounts as accounts;
+use accounts::REGISTRAR_ROLE;
+
+mod default_weight;
+#[cfg(test)]
+mod mock;
+#[cfg(feature = "payment")]
+mod payment;
+#[cfg(test)]
+mod tests;
+
+//Not sure, are those derives required?
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+#[derive(Encode, Decode, Clone, PartialEq, Eq, Debug)]
+pub enum ZoneType {
+    /// Forbidden type zone
+    Red,
+    /// Available for safe flights
+    Green,
+    /// Owns zones
+    Parent,
```

Comment:
Can't see how is it used now. May be it is not ideal from design perspective to mix "available/unavailable" with "owner/leaf" in one enum, might be SRP issue. Let's discuss if unclear.